<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>NvidiaDeepgrowSegTool2DGUI</class>
  <widget class="QWidget" name="NvidiaDeepgrowSegTool2DGUI">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>320</width>
        <height>240</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <item>
        <widget class="QLabel" name="label">
          <property name="text">
            <string>Press (Shift + Click) to add points. Change server URI in Nvidia AIAA preferences (Ctrl+P)</string>
          </property>
          <property name="wordWrap">
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QComboBox" name="deepgrowCombo">
          <property name="toolTip">
            <string>Select Your Deepgrow Model</string>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QGridLayout" name="serverSettings">
          <item row="0" column="0">
            <widget class="QPushButton" name="backgroundBtn">
              <property name="toolTip">
                <string>Switch between foreground/background to add points</string>
              </property>
              <property name="text">
                <string>Background</string>
              </property>
              <property name="checkable">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item row="0" column="1">
            <widget class="QPushButton" name="clearPointsBtn">
              <property name="toolTip">
                <string>Click to Discard All Selected Points</string>
              </property>
              <property name="text">
                <string>Clear Points</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <resources />
  <connections />
</ui>
